asab/web/auth/service.py

```diff
@@ -171,14 +164,25 @@ def is_enabled(self) -> bool:
 		return self.Mode in {AuthMode.ENABLED, AuthMode.MOCK}
 
 
-	def install(self, web_container):
+	def install(self, web_container=None):
 		"""
 		Apply authorization to all web handlers in a web container, according to their arguments and path parameters.
 
 		:param web_container: Web container to be protected by authorization.
 		:type web_container: asab.web.WebContainer
 		"""
-		# TODO: Call this automatically if there is only one container
+		if web_container is None:
+			# Locate web container if there is only one
+			web_service = self.App.get_service("asab.WebService")
+			if len(web_service.Containers) != 1:
+				return
```

Comment:
There should be a warning, not a silent fault.

Reply:
i wanted to avoid logging a warning every time i start a service with more than one webcontainer (e.g. seacat auth).

i made a dedicated method for auto-installation instead so that `install` does not have this deceitful condition.

@ateska maybe i could instead check in `async initialize` if authorization has been installed to at least one web container and log a warning if it has not?